### PR TITLE
Update Teleport command

### PR DESCRIPTION
**Pull request by Pablete1234**
Prevent tp outside of the world
Localize teleport message
Add teleported by mesage

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java [hunk 1]
```diff
@@ -69,8 +89,27 @@ public static void teleport(final CommandContext cmd, CommandSender sender) thro
...
                         if (cmd.getString(3).contains("~")) z += teleporting.getLocation().getZ();
-                        teleporting.teleport(new Location(teleporting.getWorld(), x, y, z, teleporting.getLocation().getYaw(), teleporting.getLocation().getPitch()));
-                        sender.sendMessage(ChatColor.YELLOW + "Teleported.");
+                        if ( -30000000 < x && x < 30000000) {
+                            if ( -30000000 < z && z < 30000000) {
+                                teleporting.teleport(new Location(teleporting.getWorld(), x, y, z, teleporting.getLocation().getYaw(), teleporting.getLocation().getPitch()));
+                                sender.sendMessage(ChatColor.YELLOW + new LocalizedChatMessage(ChatConstant.GENERIC_TELEPORTED).getMessage(((Player) sender).getLocale()));
+                                if ((Player) sender != teleporting) {
+                                teleporting.sendMessage(ChatColor.YELLOW + new LocalizedChatMessage(ChatConstant.GENERIC_TELEPORTED_BY, ((Player) sender).getDisplayName()).getMessage((teleporting.getLocale())));
```
Comment (iEli2tyree011) [inline src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java]: `if` block isn't spaced properly

File: src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java [hunk 2]
```diff
@@ -69,8 +89,27 @@ public static void teleport(final CommandContext cmd, CommandSender sender) thro
...
+                                numTooBig.addWith("30000000");
+                                numTooBig.setColor(net.md_5.bungee.api.ChatColor.RED);
+                                ((Player) sender).sendMessage(numTooBig);
+                            }
+                        } else {
+                            TranslatableComponent numTooBig = new TranslatableComponent("commands.generic.num.tooBig");
+                            numTooBig.addWith("" + x);
+                            numTooBig.addWith("30000000");
+                            numTooBig.setColor(net.md_5.bungee.api.ChatColor.RED);
+                            ((Player) sender).sendMessage(numTooBig);
```
Comment (iEli2tyree011) [inline src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java]: If this command uses `(Player) sender` so much we can probably just make it a local variable.

**Comment by Pablete1234** [comment]
@iEli2tyree011 fixed

File: src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java [hunk 3]
```diff
@@ -21,14 +22,15 @@
     @Command(aliases = {"tp", "teleport"}, desc = "Teleport players.", usage = "<player> [player], [player] [[~]x] [[~]y] [[~]z]", min = 1, max = 4)
     public static void teleport(final CommandContext cmd, CommandSender sender) throws CommandException {
         if (sender instanceof Player) {
+            Player SendingPlayer = (Player) sender;
```
Comment (iEli2tyree011) [inline src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java]: Never start variables like this with an uppercase letter. Perhaps just call this variable `player` and if `player` is already being used somewhere else, then change the other variables to `target` or something similar.

**Comment by Pablete1234** [comment]
@iEli2tyree011 done

**Comment by Pablete1234** [comment]
Anything else that should be added?

**Comment by twizmwazin** [comment]
merged

**Pull request closed without merging**